microsoft-edge/web-platform/user-agent-guidance.md

```diff
@@ -93,31 +114,41 @@ If you need to access more detailed information, use the [getHighEntropyValues()
    :::column-end:::
 :::row-end:::  
 
-## Recommended uses of User-Agent Client Hints  
+For more information, navigate to [getHighEntropyValues()][GithubWicgUaClientHintsGethighentropyvalues].
 
-The set of brands and the associated display order change over time, so you should never hard-code indices into the array of returned brands.  To help ensure you spot similar issues in your website early, the browser includes a `GREASE` brand value that changes over time and is formatted to break because of common string parsing issues.  
+### User-Agent Client Hints suggested use
 
-If you're prevented from using [feature detection][MdnLearnToolsTestingCrossBrowserTestingFeatureDetection], don't use a hardcoded list of known Chromium-based browsers for verification.  Examples of hardcoded browser names include `Microsoft Edge` and `Google Chrome`.  Reasons why [feature detection][MdnLearnToolsTestingCrossBrowserTestingFeatureDetection] isn't applicable may include the following situation.  
+Combining User-Agent Client Hints with [feature detection][MdnLearnToolsTestingCrossBrowserTestingFeatureDetection] is an effective way to deliver compatible web content. Microsoft recommends using this pattern to:
+* Improve code maintainability.
+* Reduce code fragility.  
+* Reduce code breakage from changes to the User-Agent String.
 
-*   A fix for a Chromium bug in a later version must be avoided and the affected browsers are difficult to detect outside of a verification of the brand and version.  
-    
-Instead, you should verify the `Chromium` brand, which ensures your detection applies to all affected Chromium-based browsers.  
+If you need to check for a Chrome-like browser, Microsoft recommends detecting `Chromium`; the engine that powers both Microsoft Edge and Google Chrome.
 
+Use this method to verify the `Chromium` brand and apply detection to all affected Chromium-based browsers.
 
 ```javascript
 function isChromium() {
     for (brand_version_pair in navigator.userAgentData.brands) {
-        if (brand_version_pair.brand == "Chromium") {
+        if (brand_version_pair.brand == "Chromium"){
             return true;
         }
     }
     return false;
 }
-```  
+```
+
+Display orders of brand names change over time. Avoid hard-coding checks for brands at specific indices.

```

Comment:
```suggestion
Use the above method to avoid hard-coding checks for brands at specific indices. Display orders of brand names can change over time. 
```